### PR TITLE
pyproject.toml: clarify build system version

### PR DESCRIPTION
**Pull request by adamjstewart**
See https://github.com/jupyter/jupyter-packaging/issues/120

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["jupyter_packaging~=0.9,<2", "jupyter_server"]
+requires = ["jupyter_packaging~=0.9", "jupyter_server"]
```
Comment (blink1073) [inline pyproject.toml:2]:
```suggestion
requires = ["jupyter_packaging>0.9,<2", "jupyter_server"]
```

**Review by blink1073 — APPROVED** [review]

**Comment by blink1073** [comment]
Thanks!  I updated it to include 1.x support.

**Comment by adamjstewart** [comment]
Was dropping 0.9 support intentional?

**Comment by blink1073** [comment]
I was looking at https://github.com/jupyter/jupyter-packaging/issues/120 when I made the change.  It doesn't matter very much, since the library is only used in the isolated build environ.

**Pull request merged**